src/services/expenses.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { getNewId } = require('../utils/getNewId');
+
+let expenses = [];
+
+const filterExpenses = (filters) => {
+  const { userId, categories, from, to } = filters;
+
+  let tempExpenses = [...expenses];
+
+  if (userId) {
+    tempExpenses = tempExpenses.filter(expense => (
+      String(expense.userId) === userId
```

Comment:
would be better to compare Number
```suggestion
      expense.userId === Number(userId)
```